independent-publisher-connectors/FreeAgent/apiProperties.json

```diff
@@ -0,0 +1,82 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "Ig_9msmfd2ua93ZVxeO65w",

```

Comment:
@nk-gears Please remove the client id value as per Microsoft security policies. You can replace with a dummy text, like below:

`"clientId": "[[CLIENT ID]]"` or `"clientId": "[[DUMMY]]"`

Reply:
@sriyen-msft Fixed and Committed.